cli/examples/v2/all.json

```diff
@@ -0,0 +1,21 @@
+{
+  "libVersion": 2,
+  "name": "All Rule Set",
+  "owner": "3fQZbDAfryjoH8GyRe4VUYAixkyQVZnQu5xD2PbV8KeU",
+  "operations": {
+    "Transfer:Holder": {
+      "type": "All",
+      "rules": [
+        {
+          "type": "Pass"
+        },
+        {
+          "type": "Amount",
+          "field": "Amount",
+          "operator": "=",
+          "amount": "01"
```

Comment:
Shouldn't this be a number? (Same in other amount rule examples for v2)

Reply:
They are a number, not sure why they show up as string once you export the JSON. The files were generated using the `print` command after converting the revision to V2.

Reply:
The `print` was run with an outdated sdk in the example above. It is working fine with the latest sdk.